packages/entities/entities-plugins/src/components/free-form/RequestCallout/types.ts

```diff
@@ -88,8 +88,10 @@ export interface RedisClusterNode {
  *                    Callout                   *
  ************************************************/
 
+export const CalloutId: unique symbol = Symbol()
```

Comment:
nit: Maybe we can also give this Symbol a description to allow a better debugging experience in the future